### DISCUSSION TITLE
What is the scientific name for the common domestic cat 🐈  ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#26] [Galaxy Qs] [answered]
What is the scientific name for the common domestic cat ?

**Comment by zolagonano** [comment, marked as answer]
The scientific name for the common domestic cat is [Felis catus](https://www.fws.gov/species/domestic-cat-felis-catus), and It is widely used in the scientific community to refer to this species.

**Reply by Sriansh-raj** [reply]
@zolagonano 🎉 answer one more question to get the `Galaxy Brain` achievement.
